### PR TITLE
chore: move dependencies to devdependencies

### PR DESCRIPTION
**Pull request by vcapretz**
just to separate each dependency so we can know if it's suposed to be a
dependency or not

## Description

Closes #XXX 

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -15,6 +15,30 @@
     "@storybook/addons": "3.3.10",
     "@storybook/react": "3.3.10",
     "@storybook/storybook-deployer": "2.2.0",
```
Comment (thiagommedeiros) [inline webapp/package.json]: I guess the storybook and enzyme is a dev dependency, or not?

Comment (vcapretz) [reply webapp/package.json]: nope, when you add it as devDependencies and run the project they will tell you that they need to be usual dependencies (although I don't understand why)

Comment (thiagommedeiros) [reply webapp/package.json]: maybe it's a bug, look that: https://github.com/storybooks/storybook/issues/1992

**Comment by vcapretz** [comment]
done @thiagommedeiros

File: webapp/package.json (hunk 1 above)
Comment (Hypnosphi) [reply webapp/package.json]: It's not a bug, it's how [`import/no-extraneous-dependencies`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md) rule works. You need additional configuration to allow importing `devDependencies` to some files

Comment (Hypnosphi) [reply webapp/package.json]: Weird, you actually have that configured: https://github.com/pagarme/artis/blob/master/.eslintrc#L20

@vcapretz can you please share the text of the error you get?

**Pull request merged**